apis/expenses-api-express/v1.0/README.md

```diff
@@ -0,0 +1,56 @@
+![Auth0 Open Source Software](https://cdn.auth0.com/resources/oss-source-large-2x.png)
+
+# Expenses API Express - The Expenses API is a simple api that is intended to be used as a starting point for hands on labs.
+
+## Environment Variables
+
+| Variable           | Value                                | Config           | Vercel Only | Default |
+| ------------------ | ------------------------------------ | ---------------- | ----------- | ------- |
+| ISSUER_BASE_URL    | https://your-tenant.region.auth0.com | issuerBaseUrl    | ❌          | ❌      |
+| ALLOWED_AUDIENCES  | **https://expenses-api**             | allowedAudiences | ❌          | ✅      |
+| VERCEL_URL         | value supplied by Vercel             |                  | ✅          | ✅      |
```

Comment:
Do we need to include the VERCEL specific envars here? For local development, they are not used. I could go either way on this.